docs/DeveloperGuide.md

```diff
@@ -505,38 +516,38 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case ends.
 
 
-**Use case: Importing data from .csv file**
+**Use case: UC 10. Importing data from [CSV](#glossary) file**
 
 **MSS**
 1. User has a data file with resident’s information, and wants to view it in RC4HDB.
-2. User uses the import command to import the CSV file.
-3. RC4HDB reads the file stored in the file path entered by the user.
-4. RC4HDB displays the name of the csv file after the file has been read.
+2. User imports the file.
```

Comment:
Now that I think about it, we could potentially combine 1 and 2. This is because if we desire to show the intention of the user, maybe it should be "User wants to switch to a separate CSV file with resident information".

Reply:
Actually, I'm inclined to keep both steps. Because Step 1 talks about the intent and when the Use Case holds, while Step 2 is the action the User does. Let me know what you think!

Reply:
Yup, no problem, it works either way! 👍